utils/common.py

```diff
@@ -39,3 +39,14 @@ def wrapper(k, *args, **kwargs):
         setattr(k, "make_object", eval("make_object", *args, **kwargs))
         return k
     return wrapper
+
+
+def format_timedelta(td):
```

Comment:
Can't we simply use `str(td)`? For example, I get a clean formatted output for the following:
```
>>> import datetime
>>> start = datetime.datetime(2018, 6, 20, 11, 0)
>>> end = datetime.datetime(2018, 6, 20, 12, 9)
>>> delta = end - start
>>> str(delta)
'1:09:00'
```

Reply:
Yes, that should work too. Let me check